pkgs/applications/version-management/git-and-tools/git-open/default.nix

```diff
@@ -0,0 +1,29 @@
+{stdenv, git, fetchFromGitHub, ...}:
```

Comment:
Nit: please do not use `...` in package expressions; they should normally declare all required params and nothing more.